test/tests/ver-1ia/comparison_ver-1ia.py

```diff
@@ -0,0 +1,77 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+from scipy import special
+import os
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+script_folder = os.path.dirname(__file__)
+os.chdir(script_folder)
+
+#===============================================================================
+# Physical constants
+kb = 1.380649e-23  # J/K Boltzmann constant
+
+# ===============================================================================
+# Extract TMAP8 results
+if "/TMAP8/doc/" in script_folder:     # if in documentation folder
+    csv_folder = "../../../../test/tests/ver-1ia/gold/ver-1ia_out.csv"
+else:                                  # if in test folder
+    csv_folder = "./gold/ver-1ia_out.csv"
+tmap8_solution = pd.read_csv(csv_folder)
+tmap8_solution_time = tmap8_solution['time']
+tmap8_solution_D2 = tmap8_solution['pressure_D2']
+tmap8_solution_H2 = tmap8_solution['pressure_H2']
+tmap8_solution_HD = tmap8_solution['pressure_HD']
```

Comment:
```suggestion
tmap8_solution_A2 = tmap8_solution['pressure_A2']
tmap8_solution_B2 = tmap8_solution['pressure_B2']
tmap8_solution_AB = tmap8_solution['pressure_AB']
```

Reply:
You need to be consistently using A, B instead of H and D. Right now you are using a mix of them. Please apply these suggestions to all the other cases.